src/Workspaces/Core/Portable/NamingStyles/Serialization/SymbolSpecification.cs

```diff
@@ -60,6 +33,36 @@ internal class SymbolSpecification
             ApplicableSymbolKindList = symbolKindList;
         }
 
+        public static SymbolSpecification CreateDefaultSymbolSpecification()
+        {
+            return new SymbolSpecification(
+                id: Guid.NewGuid(),
```

Comment:
doc that this gives a fresh guid each time, which is what we want. (i assume that's true)